src/main/resources/assets/iris/lang/fr_fr.json

```
@@ -0,0 +1,5 @@
+{
+  "iris.shaders.reloaded": "Shaders Rechargés!",
+  "iris.keybind.reload": "Recharger Les Shaders",
+  "iris.shaders.reloaded.failure": "Echec lors du rechargement des shaders! Raison: %s"
```

Upper case e of Echec needs an accent
```suggestion
  "iris.shaders.reloaded.failure": "Échec lors du rechargement des shaders! Raison: %s"
```